database/seed.js

```diff
@@ -86,6 +114,25 @@ const seedDatabase = async () => {
     const document = new Item({ ...item, owner: user._id })
     await document.save()
   }
+
```

Comment:
@phreid can I seed the db with this data? I think my thunk isn't working because axios doesn't recognize '/requests/' as a valid URL

Reply:
it looks like you haven't set up express to use your requests route. see the index.js file that starts the server.